cli/types/cypress.d.ts

```diff
@@ -2556,6 +2558,8 @@ declare namespace Cypress {
      *    expect(s).to.have.been.calledOnce
      */
     withArgs(...args: any[]): Omit<A, 'withArgs'> & Agent<A>
+
+    callsFake(func: (...args: any[]) => any): Omit<A, 'withArgs'> & Agent<A>
```

Comment:
There's a lot of methods that aren't extended from Sinon here. I added this one because our code was erroring trying to chain `.as()` off of `cy.stub().callsFake()`